acme/common.go

```diff
@@ -22,17 +22,6 @@ type Identifier struct {
 	Value string `json:"value"`
 }
 
-var (
-	accountTable           = []byte("acme-accounts")
-	accountByKeyIDTable    = []byte("acme-keyID-accountID-index")
-	authzTable             = []byte("acme-authzs")
-	challengeTable         = []byte("acme-challenges")
-	nonceTable             = []byte("nonce-table")
```

Comment:
You've renamed it to nonces.

Reply:
And you're using `_` instead of `-`

Reply:
Yeah, so using `_` instead of `-` because that's turns out to be a sql special character. All the original tables I created (for certs, revocations, otts) used `_`s. I updated nosql to use backquote all table names anyway (so illegal chars shouldn't matter), but I'd rather not use them when possible, and in this case it's more consistent with the original table names anyway.

Reply:
ok